.github/workflows/review-instance.yml

```
@@ -0,0 +1,35 @@
+name: Review Instance
+on: 
+  workflow_dispatch:
+    inputs:
+      devops_branch:
+        description: 'devops branch'     
+        required: true
+        default: 'master'
+      api_branch:
+        description: 'vets-api branch'
+        required: true
+        default: 'main'
```

```suggestion
        default: 'master'
```
vets-api still uses the `master` branch as its default